src/components/admin/Players.vue

```diff
@@ -117,7 +126,19 @@ export default {
       });
     },
     deletePlayer: function (playerId) {
-      this.apiDeletePlayer(playerId).then(() => {
+      this.apiDeletePlayer(playerId).then((data) => {
+        if (data.deletePlayer) {
+          this.$notify({
+            group: "success",
+            type: "success",
+            title: this.$t("playerDeleted"),
+            text: this.$t("msgPlayerDeleted", {
+              player: this.players.filter(function (player) {
+                return player.id === playerId;
+              })[0].name,
```

Comment:
Our `delete` mutations could send back the deleted data if the operation is successful. You wouldn't need to look for the deleted player in the list.